integration/tests/existing_vpc/existing_vpc_test.go

```diff
@@ -0,0 +1,207 @@
+//go:build integration
+// +build integration
+
+package unowned_clusters
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/weaveworks/eksctl/pkg/eks"
+
+	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+
+	. "github.com/weaveworks/eksctl/integration/runner"
+	"github.com/weaveworks/eksctl/integration/tests"
+	"github.com/weaveworks/eksctl/pkg/testutils"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var params *tests.Params
+
+func init() {
+	// Call testing.Init() prior to tests.NewParams(), as otherwise -test.* will not be recognised. See also: https://golang.org/doc/go1.13#testing
+	testing.Init()
+	params = tests.NewParams("exist-vpc")
+}
+
+func TestVPC(t *testing.T) {
+	testutils.RegisterAndRun(t)
+}
+
+var _ = Describe("(Integration) [using existing VPC]", func() {
+	var (
+		stackName  string
+		ng1        = "ng-1"
+		mng1       = "mng-1"
+		ctl        api.ClusterProvider
+		configFile *os.File
+		cfg        *api.ClusterConfig
+	)
+
+	BeforeSuite(func() {
+		stackName = fmt.Sprintf("eksctl-%s", params.ClusterName)
+		cfg = &api.ClusterConfig{
+			TypeMeta: api.ClusterConfigTypeMeta(),
+			Metadata: &api.ClusterMeta{
+				Name:   params.ClusterName,
+				Region: params.Region,
+			},
+		}
+
+		var err error
+		configFile, err = ioutil.TempFile("", "")
+		Expect(err).NotTo(HaveOccurred())
+
+		clusterProvider, err := eks.New(&api.ProviderConfig{Region: params.Region}, cfg)
+		Expect(err).NotTo(HaveOccurred())
+		ctl = clusterProvider.Provider
+		cfg.VPC = createVPC(stackName, ctl)
+
+		configData, err := json.Marshal(&cfg)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(ioutil.WriteFile(configFile.Name(), configData, 0755)).To(Succeed())
+		cmd := params.EksctlCreateCmd.
+			WithArgs(
+				"cluster",
+				"--config-file", configFile.Name(),
+				"--verbose", "2",
+			).WithoutArg("--region", params.Region)
+		Expect(cmd).To(RunSuccessfully())
+	})
+
+	AfterSuite(func() {
+		// deleteStack(stackName, ctl)
+		Expect(os.RemoveAll(configFile.Name())).To(Succeed())
+	})
+
+	It("supports creating managed and unmanaged nodegroups in the existing VPC", func() {
+		cfg.NodeGroups = []*api.NodeGroup{{
+			NodeGroupBase: &api.NodeGroupBase{
+				Name: ng1,
+			}},
+		}
+		cfg.ManagedNodeGroups = []*api.ManagedNodeGroup{{
+			NodeGroupBase: &api.NodeGroupBase{
+				Name: mng1,
+			}},
+		}
+		// write config file so that the nodegroup creates have access to the vpc spec
```

Comment:
if you add more `By`s as part of the other review comment, you could also change these doc comments into `By`s as well (super nit! feel free to ignore)